@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fuselage/fn_fuselageDrag.sqf

```diff
@@ -8,19 +11,41 @@ private _fuselageAreaBottom = _heli getVariable "fza_sfmplus_fuselageAreaBottom"
 
 private _fuselageDragCoefX    = 1.5;
 private _fuselageDragCoefZ    = 0.5;
-//                                  PA   -40     0    40
-private _dragCoefTableY       = [[   0, 0.54, 0.60, 0.60],   //ft 
-                                 [2000, 0.80, 0.70, 0.55],   //ft
-                                 [4000, 1.02, 0.92, 0.85],   //ft
-                                 [6000, 1.20, 1.00, 1.06],   //ft
-                                 [8000, 1.65, 1.25, 1.70]];  //ft
-
-private _interpDragCoefTableY = [_dragCoefTableY, _altitude] call fza_fnc_linearInterp;
-_dragCoefTableY               = [[-40, _interpDragCoefTableY # 1],
-                                 [  0, _interpDragCoefTableY # 2],
-                                 [ 40, _interpDragCoefTableY # 3]];
-_interpDragCoefTableY         = [_dragCoefTableY, _temperature] call fza_fnc_linearInterp;
-private _fuselageDragCoefY    = _interpDragCoefTableY # 1;
+
+private _interpDragCoefTableY = [];
+
+if (_flightModel == "SFMPlus") then {
+    //                                  PA   -40     0    40
+    private _sfmPlusdragCoefTableY = [[   0, 0.54, 0.60, 0.60]   //ft 
+                                     ,[2000, 0.80, 0.70, 0.55]   //ft
```

Comment:
Please can this be moved to config?

Reply:
Until it's fully finished i'd prefer to leave it here for now, it's easier to edit and adjust.